designer/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/ProcessActionEntityFactory.scala

```diff
@@ -92,5 +92,6 @@ final case class ProcessActionEntityData(
 
   lazy val createdAtTime: Instant           = createdAt.toInstant
   lazy val performedAtTime: Option[Instant] = performedAt.map(_.toInstant)
-  lazy val isDeployed: Boolean              = actionType.equals(ProcessActionType.Deploy)
+  lazy val isDeployed: Boolean =
+    actionType.equals(ProcessActionType.Deploy) && !state.equals(ProcessActionState.ExecutionFinished)
```

Comment:
perhaps this change should be reflected in some UT (i think the only changes in tests in this PR are renames)

Reply:
let's use `==` instead of `equals`, we are in scala, not some prehistoric java! ;-)   
W8 I can't see usage of that. It looks like a death code. Isn't it?

Reply:
Yes yes yes! All undocumented as externally used death code should be always removed. I even looked for some tool for detecting that but didn't find any reliable one :(